dev/automation-hub/initial_data.json

```diff
@@ -33,7 +33,7 @@
       "pulp_last_updated": "2020-03-12T15:56:19.072Z",
       "pulp_type": "ansible.ansible",
       "name": "automation-hub",
-      "description": null,
+      "description": "Default golden repository, the source of downloads",
```

Comment:
I'd say avoid the use of 'golden' for anything might be customer facing. We kind of know what it means but not sure customers do.

Reply:
updated